pinot-plugins/pinot-stream-ingestion/pinot-kinesis/pom.xml

```diff
@@ -36,9 +36,26 @@
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
     <phase.prop>package</phase.prop>
-    <aws.version>2.15.50</aws.version>
+    <aws.version>2.14.28</aws.version>
+    <powermock.version>2.0.9</powermock.version>
```

Comment:
remove the powermock if it is not being used anymore

Reply:
resolved.